src/docs/asciidoc/tcp-client.adoc

```diff
@@ -302,11 +302,15 @@ public class Application {
 
 == Connection Pool
 
-By default, the `TCP` client uses an "`elastic`" connection pool. This means that the implementation creates a new channel
-if someone tries to acquire a channel but none is in the pool. Also, there is no limit on the maximum concurrent channels.
+By default, the `TCP` client uses a "`fixed`" connection pool with `500` as a maximum number of the channels and
+`45s` as acquisition timeout.
+This means that the implementation creates a new channel if someone tries to acquire a channel but none is in the pool.
+When the maximum number of the channels in the pool is reached, new tries to acquire a channel will be delayed
+until a channel is returned to the pool again.
 The implementation uses `FIFO` order for channels in the pool regardless of whether it is "`elastic`" or "`fixed`".
+By default there is no idle time specified for the channels in the pool
```

Comment:
Please add a comma after "default" and a period at the end.

Reply:
done